src/Aspire.Hosting/Dashboard/proto/dashboard_service.proto

```diff
@@ -54,8 +54,8 @@ message ResourceCommand {
 }
 
 enum IconVariant {
-    REGULAR = 0;
-    FILLED = 1;
+    ICON_VARIANT_REGULAR = 0;
+    ICON_VARIANT_FILLED = 1;
 }
```

Comment:
FYI the proto buff enum use C++ scoping rules, which means they each have to be distrinct. This isn't a breaking change on the wire (only the id matters) and it isn't a breaking change for code gen which removes the prefix.